task/ecosystem-cert-preflight-checks/0.2/ecosystem-cert-preflight-checks.yaml

```diff
@@ -76,9 +76,21 @@ spec:
           exit 0
         fi
 
+        # If the image URL points to a manifest list (a multi-arch image), check the labels on any of the child
+        # images (don't fail in the case where the list does not include an image for the arch of the system
+        # where this pipeline is running).
+        _INSPECT_OVERRIDE_IMAGE_ARCH=
+        _INSPECT_OVERRIDE_IMAGE_OS=
+        _RAW_IMAGE_MANIFEST=$(skopeo inspect --raw "docker://${PARAM_IMAGE_URL}")
+        _IMAGE_MEDIA_TYPE=$(printf "%s" "${_RAW_IMAGE_MANIFEST}" | jq -r '.mediaType')
+        if [[ "${_IMAGE_MEDIA_TYPE}" == "application/vnd.docker.distribution.manifest.list.v2+json" ]]; then
```

Comment:
You probably need to also match the OCI Index mediatype, as the pipelines (and tooling) seem to default to it. I see it here: https://specs.opencontainers.org/image-spec/image-index/

Reply:
Might also want to log the media type if it's not already being emitted somewhere, just for debugging.